config/allowed_functions.txt

```diff
@@ -0,0 +1,282 @@
+abbrev
```

Comment:
Why are we overriding this file? Which function(s) do we want to add or exclude?

Reply:
This file was referred from the config when we got our copy or your environment. We've just added it to the repo, so that it won't get lost accidentally. Try asking your co-workers, why they added it initially

Reply:
Wait, do you mean a copy of a live environment? We have plenty of config there that isn't relevant to this particular extension...